sections/products.liquid

```diff
@@ -0,0 +1,103 @@
+{{ 'products.css' | asset_url | stylesheet_tag }}
+
+<div class="products-section">
+  {% for i in (1..4) %}
+    {% assign product_id = 'product_' | append: i %}
+    {% assign product = all_products[section.settings[product_id]] %}
+
+    {% if product %}
+      <div class="product" >
+        <div class="product__img-container">
+          <img
+            src="{{ product.images[0] | img_url: 'master' }}"
+            width="204"
+            alt="{{ product.title }}"
+            class="product__img"
+          >
+          <p class="product__qualification">
+            <svg
+              width="16"
+              height="16"
+              viewBox="0 0 16 16"
+              class="card__star-icon"
+              fill="none"
+              xmlns="http://www.w3.org/2000/svg"
+            >
+              <path id="Vector" d="M16 6.17662L10.1875 5.80159L7.99375 0.298462L5.79997 5.80159L0 6.17662L4.44997 9.96409L2.98747 15.7016L7.99375 12.5328L13 15.7016L11.5375 9.96409L16 6.17662Z"fill="#FDBF00" />
+            </svg>
```

Comment:
You can create a snippet with the svg code and render it here instead. For instance, you can create a `star-icon.liquid` file in `/snippets` and include it as in the suggestion below

```suggestion
            {% render 'star-icon' %}
```